hack/Dockerfile.installer-frontend

```diff
@@ -0,0 +1,47 @@
+FROM installer as build
+ARG TARGETARCH
+ENV TARGETARCH ${TARGETARCH}
+RUN apk add --no-cache --update \
+    cpio \
+    squashfs-tools \
+    xz
+WORKDIR /initramfs
+ARG RM
+RUN <<EOF
+set -euo pipefail
+
+xz -d /usr/install/${TARGETARCH}/initramfs.xz
+cpio -idvm < /usr/install/${TARGETARCH}/initramfs
+unsquashfs -f -d /rootfs rootfs.sqsh
+
+for f in ${RM}; do
```

Comment:
do we need this, we can always assume to remove `/lib/modules` when using a custom installer?